src/ImmutableX.ts

```diff
@@ -592,19 +592,17 @@ export class ImmutableX {
 
   /**
    * Completes a Withdrawal
-   * @param ethSigner - the L1 signer
-   * @param starkPublicKey - the Signer address
+   * @param walletConnection - the pair of L1/L2 signers
    * @param token - the token
    * @returns a promise that resolves with the transaction
    * @throws {@link index.IMXError}
    */
   public completeWithdrawal(
-    ethSigner: EthSigner,
-    starkPublicKey: string,
+    walletConnection: WalletConnection,
     token: AnyToken,
```

Comment:
seems that this is a breaking change and thus, maybe we need to either bump the version number to a major version (eg: 3.0.0) or we need to make sure the code is backwards compatible, thoughts?